src/AppBundle/AppBundle.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /*
```

Comment:
i guess we have been neglectant on these. yeah lets have them everywhere as usual.